primitives/runtime/src/messages.rs

```diff
@@ -36,6 +36,12 @@ pub enum DispatchFeePayment {
 /// Message dispatch result.
 #[derive(Encode, Decode, RuntimeDebug, Clone, PartialEq, Eq)]
 pub struct MessageDispatchResult {
+	/// Dispatch result flag. This flag is relayed back to the source chain and, generally
+	/// speaking, may bring any (that fits in single bit) information from the dispatcher at
+	/// the target chain to the message submitter at the source chain. If you're using immediate
+	/// call dispatcher, then it'll be result of the dispatch - `true` if dispatch has succeeded
+	/// and `false` otherwise.
+	pub dispatch_result: bool,
```

Comment:
```suggestion
	pub dispatched: bool,
```
I find it a bit confusing to have `dispatch_result` and `bool` - it kind of signifies that there is some extra result embedded in case that's true.

I'd call this simply `dispatched: true/false` cause it's part of `MessageDispatchResult` struct already. It's just a personal preference, there is nothing wrong really with `dispatch_result`, so if you don't find my arguments convincing feel free to leave as-is.